examples/slides/template.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
```

Comment:
Could you please add the command to generate the pdf in the README ?

Reply:
done